charger/eebus.go

```diff
@@ -42,18 +42,26 @@ func init() {
 // NewEEBusFromConfig creates an EEBus charger from generic config
 func NewEEBusFromConfig(other map[string]interface{}) (api.Charger, error) {
 	cc := struct {
-		Ski string
-	}{}
+		Ski           string
+		Ip            string
+		Meter         bool
+		ChargedEnergy bool
+	}{
+		Meter:         true,
```

Comment:
Hier kein Default, dann kann brauchst Du im Template auch kein explizites `false` und das `eebusmeter` in der `default.yaml`.